### PR TITLE
Web Components: Reintegrate `@storybook/lit` into `@storybook/web-components`

### PR DESCRIPTION
**Pull request by gaetanmaisse**
## What I did

 - Clean `@storybook/web-components` dependencies and Webpack config
 - Update `web-components-kitchen-sink` with components and stories developed in `lit-kitchen-sink`
 - Delete `lit-kitchen-sink`
 - Delete `@storybook/lit` in favor of `@storybook/web-components`

## How to test

- CI should be 🟢 
- Locally you should be able to run `yarn install && yarn storybook` in `web-components-kitchen-sink`

### CODE REVIEW

File: app/web-components/src/server/framework-preset-web-components.ts [hunk 1]
```diff
@@ -1,39 +1,5 @@
...
-              useBuiltIns: 'entry',
-              corejs: 3,
-            },
-          ],
-        ],
-        babelrc: false,
-      },
-    },
-  });
-
```
Comment (gaetanmaisse) [inline app/web-components/src/server/framework-preset-web-components.ts:37]: @ndelangen @shilman I would like your feedback about that, I'm not sure if we should still keep it or not

Comment (shilman) [reply app/web-components/src/server/framework-preset-web-components.ts:37]: I'm not sure either. Perhaps consult the WC guys? Or @brion-fuller ?

Comment (brion-fuller) [reply app/web-components/src/server/framework-preset-web-components.ts:37]: We will need to keep this for compatibility with webpack 4.  Webpack 4 doesn't support `import.meta`.

Comment (gaetanmaisse) [reply app/web-components/src/server/framework-preset-web-components.ts:37]: Ok, I will discard this commit then, thanks @brion-fuller

**Comment by shilman** [comment]
Looks funny?
Example___Button_-_Secondary_⋅_Storybook

**Review by ndelangen — APPROVED** [review]

**Review by shilman — APPROVED** [review]
Great job @gaetanmaisse !!! 🙌

**Pull request merged**